recipes/vtk-cdat/build.sh

```diff
@@ -0,0 +1,160 @@
+        -DModule_vtkInteractionWidgets:BOOL=ON \
+        -DModule_vtkIOCore:BOOL=ON \
+        -DModule_vtkIOExport:BOOL=ON \
+        -DModule_vtkIOExportOpenGL:BOOL=ON \
+        -DModule_vtkIOGeometry:BOOL=ON \
+        -DModule_vtkIOImage:BOOL=ON \
+        -DModule_vtkIOImport:BOOL=ON \
+        -DModule_vtkRenderingCore:BOOL=ON \
+        -DModule_vtkRenderingFreeType:BOOL=ON \
+        -DModule_vtkRenderingFreeTypeOpenGL:BOOL=ON \
+        -DModule_vtkRenderingImage:BOOL=ON \
+        -DModule_vtkRenderingLabel:BOOL=ON \
+        -DModule_vtkRenderingOpenGL:BOOL=ON \
+        -DModule_vtkRenderingVolume:BOOL=ON \
+        -DModule_vtkRenderingVolumeOpenGL:BOOL=ON \
+        -DModule_vtkViewsCore:BOOL=ON \
+        -DModule_vtkViewsGeovis:BOOL=ON \
+        -DModule_vtkIOFFMPEG:BOOL=ON \
+        -DBUILD_SHARED_LIBS=ON \
+        -DVTK_WRAP_PYTHON=ON \
+        -DPYTHON_EXECUTABLE=${PYTHON} \
+        -DPYTHON_INCLUDE_PATH=${PREFIX}/include/python${PY_VER} \
+        -DPYTHON_LIBRARY=${PREFIX}/lib/${PY_LIB} \
+        -DVTK_INSTALL_PYTHON_MODULE_DIR=${SP_DIR} \
+        -DModule_vtkRenderingMatplotlib=ON \
+        -DVTK_Group_Web:BOOL=ON \
+        -DLIBPROJ4_INCLUDE_DIR:PATH=${PREFIX}/include \
+        -DVTK_RENDERING_BACKEND=OpenGL \
+        -DVTK_USE_SYSTEM_ZLIB:BOOL=ON \
+        -DVTK_USE_SYSTEM_LIBXML2:BOOL=ON \
+        -DVTK_USE_SYSTEM_HDF5:BOOL=ON \
+        -DVTK_USE_SYSTEM_NETCDF:BOOL=ON \
+        -DVTK_USE_SYSTEM_FREETYPE:BOOL=OFF \
+        -DVTK_USE_SYSTEM_LIBPROJ4:BOOL=ON \
+        -DVTK_Group_Rendering:BOOL=ON \
+        -DVTK_Group_StandAlone:BOOL=OFF \
+        -DVTK_LEGACY_SILENT:BOOL=ON"
+
+if [ ${OSNAME} == Linux ]; then
+    LIBEXT="so"
+elif [ ${OSNAME} == Darwin ]; then
+    LIBEXT="dylib"
+fi
+
+VTK_ARGS="\
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.12 \
+    -DVTK_REQUIRED_OBJCXX_FLAGS='' \
+    -DLIBPROJ4_LIBRARIES:FILEPATH=${PREFIX}/lib/libproj.${LIBEXT}"
+
+if [[ ${CONDA_LST}'y' == *'mesalib'* ]]; then
+    VTK_ARGS="${VTK_ARGS} \
+        -DVTK_USE_OFFSCREEN:BOOL=ON \
+        -DVTK_OPENGL_HAS_OSMESA:BOOL=ON \
+        -DOSMESA_INCLUDE_DIR:PATH=${PREFIX}/include \
+        -DOSMESA_LIBRARY:FILEPATH=${PREFIX}/lib/libOSMesa32.${LIBEXT}"
+
+    if [ ${OSNAME} == Linux ]; then
+        VTK_ARGS="${VTK_ARGS} \
+            -DVTK_USE_X:BOOL=OFF"
+    elif [ ${OSNAME} == Darwin ]; then
+        VTK_ARGS="${VTK_ARGS} \
+            -DVTK_USE_CARBON:BOOL=OFF \
+            -DVTK_USE_COCOA:BOOL=OFF"
+    fi
+else
+    VTK_ARGS="${VTK_ARGS} \
+        -DVTK_USE_OFFSCREEN:BOOL=OFF \
+        -DVTK_OPENGL_HAS_OSMESA:BOOL=OFF"
+    if [ ${OSNAME} == Linux ]; then
+        VTK_ARGS="${VTK_ARGS} \
+            -DVTK_USE_X:BOOL=ON"
+    elif [ ${OSNAME} == Darwin ]; then
+        VTK_ARGS="${VTK_ARGS} \
+            -DVTK_USE_CARBON:BOOL=OFF \
+            -DVTK_USE_COCOA:BOOL=ON"
+    fi
+fi
+
+COMMAND="cmake .. \
+    ${COMMON_ARGS} \
+    ${VTK_ARGS} \
+    ${MPI_ARGS}"
+echo $COMMAND
+eval ${COMMAND}
+
+make -j8
```

Comment:
No. we set it to 2 here, https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/master/recipe/run_conda_forge_build_setup_osx#L3